packages/tenacity-git/tenacity-git.pacscript

```diff
@@ -21,12 +21,12 @@ build() {
     mkdir build
     cd build
     cmake -G "Unix Makefiles" \
-        -DESTDIR="$STOWDIR/$name" \
+        -DDESTDIR="${pkgdir}" \
         -DCMAKE_BUILD_TYPE=Release \
         -Duse_ffmpeg=loaded ..
```

Comment:
```suggestion
        -DCMAKE_BUILD_TYPE=Release ..
```
try this maybe?

Reply:
I have set -DFFMPEG=OFF and it compiles now. Is the ffmpeg loading important?
I think the successor will be Saucedacity, but in current state I prefer tenacity

Reply:
I think ffmpeg loading is important, as it can handle a lot of different audio codecs.

Reply:
I figured out the aur package uses specific version of ffmpeg 4.4 and that ubuntu 22.10 updated from 4.4 to 5.1.
So I think there is no luck for kinetic users like me.
But compiles fine in jammy.
So I just say in case kinetic, compile without ffmpeg. Otherwise, compile with ffmpeg
Unless and until someone creates a ffmpeg 4.4 pacscript for kinetic, I think that will do.